src/style.css

```
@@ -1,3 +1,80 @@
-body {
+html{
+  font-family: Roboto, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+:root {
+  --blue-color: #00ACDC;
+}
+
+header {
+  box-sizing: border-box;
+  width: 100%;
+  background-color: #FFF;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 50px;
```

just use bigger value for links
```suggestion
  padding: 0 50px;
```